### ISSUE TITLE
Inline bold and italics not working in nvim 

### ISSUE THREAD

**Issue by idvorkin** [MDeiml/tree-sitter-markdown#124]
I don't get inline bold or italics (I'm guessing I'm configuring something wrong, but no clue what :( )

*For folks that don't want to read the thread:*  Install a TS compatible colorscheme. I picked: https://github.com/catppuccin/nvim

The code

```markdown
This is not bold **This is bold**
*This is italics*

```python
def  you_can_see_inline_works()
```
**Actual behavior**

![image](https://github.com/MDeiml/tree-sitter-markdown/assets/280981/01f279d2-4fa1-43b8-8108-43d82ccda6d8)

Now when I disable TS, I get what I'd expect with the default vim highlighting

![image](https://github.com/MDeiml/tree-sitter-markdown/assets/280981/ab280361-cfde-4d3b-8eeb-27c254830d47)

**Comment by idvorkin** [comment]
I can workaround this with the following settings, but it results in me losing the treesitter markdown speedups
```lua
require'nvim-treesitter.configs'.setup {
  highlight = {
    additional_vim_regex_highlighting = {'markdown','markdown_inline'},
  },
}
```

**Comment by clason** [comment]
Works for me. Make sure your color scheme sets or links the correct highlight groups (which are different from the legacy syntax groups). In any case, this is not a parser issue so OT here.

**Comment by MDeiml** [comment]
Exactly, this is possibly a problem with neovim or your specific configuration, but not with this parser.

**Comment by idvorkin** [comment]
@clason  - Thanks, I'm guessing you're right, but not sure how to debug.  Can you share the highlight groups I need to light up to verify?

**Comment by clason** [comment]
use `:Inspect`

**Comment by idvorkin** [comment]
(Sorry for the n00b questions, it's hard to find simple documentation here, feel free to link me that) 

Here's what I'm getting for inspect and inspect tree. 

Does this match what you get? Assuming it matches, how do I connect from here to neovim syntax highlighting?


![image](https://github.com/MDeiml/tree-sitter-markdown/assets/280981/ddae22db-1c99-49f6-aa67-5254acabf2a6)
```:InspectTree```
<img width="1056" alt="image" src="https://github.com/MDeiml/tree-sitter-markdown/assets/280981/6b7559ed-d033-4c2d-a6c4-88372699744d">

**Comment by clason** [comment]
You need to have the cursor on the exact text you want to `:Inspect`: (`:h Inspect`)

**Comment by idvorkin** [comment]
**TL;DR**: Install a TS compatible colorscheme. I picked: https://github.com/catppuccin/nvim

**Details**
I now understand what you're telling me. TS is creating syntax elements  correctly, but no highlight is configured for that (I'm using the vim default color schemes).  You can see this for yourself: 

Install the  [TSPlayground](https://github.com/nvim-treesitter/playground) and then ```:TSHighlightCapturesUnderCursor``` to see what the TS element was.   

You can see something similar with syntax files via: 

```vim
:echo synIDattr(synID(line("."), col("."), 1), "name")
```

You can then bind the highlight group to the TS element
```vim
highlight link @text.emphasis htmlItalic
highlight link @text.strong htmlBold
```

**Comment by idvorkin** [comment]
Sweet - thanks for the help @clason!  Also, thanks for making the awesome plugin @MDeiml! Have a great day!

**Comment by clason** [comment]
> Install the [TSPlayground](https://github.com/nvim-treesitter/playground) and then :TSHighlightCapturesUnderCursor to see what the TS element was.

`:Inspect` does the same (but better) and is built-in. (playground is deprecated)
